src/core/prompts/__tests__/system.test.ts

```diff
@@ -171,7 +171,7 @@ describe("SYSTEM_PROMPT", () => {
 	beforeEach(() => {
 		// Reset experiments before each test to ensure they're disabled by default
 		experiments = {
-			[EXPERIMENT_IDS.DIFF_STRATEGY_SEARCH_AND_REPLACE]: false,
+			[EXPERIMENT_IDS.SEARCH_AND_REPLACE]: false,
```

Comment:
This was a bit of a mixup - SEARCH_AND_REPLACE here refers to the tool, not the diff strategy